src/fmu/sumo/explorer/_explorer.py

```diff
@@ -55,8 +70,34 @@ def get_status(self):
 
         return status
 
+    def get_dict_of_cases(self, size=10):
+
+        """returns dictionary of cases where key is name, value sumo_id"""
+        results = return_hits(self.get("/searchroot", query="class:case",
+                                       select=["fmu.case.name"], size=size))
+        case_dict = {}
+        for result in results:
+            case_dict[result["_source"]["fmu"]["case"]["name"]] = result["_id"]
+        return case_dict
+
+    def get_case_by_name(self, name):
+        """Fetches case from case name
+        args:
+            name (str): name of case
+
+        """
+        query = f"class:case AND fmu.case.name:{name}"
```

Comment:
This should probably be `f"class:case AND fmu.case.name:\"{name}\""`, if `name` can contain spaces, "/", ":" or other semi-exotic characters.